lib/data/locationConstraintParser.js

```diff
@@ -12,7 +12,7 @@ export default function parseLC() {
         if (locationObj.type === 'mem') {
             clients[location] = inMemory;
         }
-        if (locationObj.type === 'file') {
+        if (locationObj.type === 'file' || locationObj.type === 'dataFile') {
```

Comment:
Without that, objects put to file before this release won't be accessible though, right?